README.md

```diff
@@ -80,4 +80,88 @@ instance applyV :: (Semigroup err) => Apply (V err)
 
 ``` purescript
 instance applicativeV :: (Semigroup err) => Applicative (V err)
+```
+
+
+
+## Module Data.Validation.Semiring
+
+
+This module defines an applicative functor and alt instance for 
+validation that can go multiple ways
+
+This validation works exactly as `Data.Validation.Semigroup`
+but uses `Semiring` instead of `Semigroup`
+
+#### `V`
+
+``` purescript
+data V err res
+```
+
+example
```

Comment:
Could you please use the style of comments in the semigroup module? You could probably just copy the text.